### PR TITLE
Fixed issue 642

### PR DESCRIPTION
**Pull request by JohnTheFarmer**
reproduced #642  with Markdown+Math installed @mamccrea

### CODE REVIEW

File: docs-preview/media/alert-styles.css [hunk 1]
```diff
@@ -1,207 +1,207 @@
...
+	text-decoration: none;
 }
 
-blockquote span {
-  display: block;
-  background-repeat: no-repeat;
-  background-position: bottom right;
-  margin-top: 1rem;
-  margin-bottom: 0;
+blockquote strong span {
```
Comment (JohnTheFarmer) [inline docs-preview/media/alert-styles.css:44]: here ;)

File: docs-preview/media/alert-styles.css [hunk 2]
```diff
@@ -1,207 +1,207 @@
...
+	display: block;
+	background-repeat: no-repeat;
+	background-position: bottom right;
+	margin-top: 1rem;
+	margin-bottom: 0;
 }
```
Comment (JohnTheFarmer) [inline docs-preview/media/alert-styles.css:50]: ;)

**Comment by bharney0** [comment]
@JohnTheFarmer Thank you for your contributions to the Docs Authoring Pack! These changes look good, just merged them. They should be deployed today or early next week.

**Pull request merged**